includes/utils.h

```diff
@@ -31,5 +31,6 @@ void						fatal_error(char const *str);
 t_uchar						get_last_exit_status(void);
 void						set_last_exit_status(t_uchar status);
 void						wait_for_childs();
+char						*get_options_core(int ac, char **av);
```

Comment:
[I already implemented a f() to retrieve options](https://github.com/42shpimanmls/42sh/blob/master/srcs/execution/builtins/opt.c).